transforms/serializable.js

```diff
@@ -0,0 +1,60 @@
+const assemblyscript = require('assemblyscript');
+const {
+  CommonFlags,
+  Node,
+  NodeKind,
+  SourceKind,
+  TypeKind,
+  parseFile
+} = assemblyscript;
+
+
```

Comment:
A general description in a comment here, of the transform, would be a good idea. 
It could offer a very simple example